.editorconfig

```diff
@@ -151,6 +151,16 @@ dotnet_public_api_analyzer.require_api_files = true
 # Workaround for https://github.com/dotnet/roslyn/issues/70570
 dotnet_diagnostic.IDE0055.severity = warning
 
+# Enable the xUnit analyzer rules
```

Comment:
Comment says enable, but this is only disabling things.

Reply:
Yes but it's referencing an issue that says enable the rules.